.github/workflows/typespec-validation-all.yaml

```diff
@@ -67,11 +67,11 @@ jobs:
         with:
```

Comment:
What does the checkout action do by default? Does it default to sparse-checkout? If not then I suspect this will not clone everything you are looking for.

Reply:
Looking at the docs I don't expect that this is a sparse-checkout but it does seem to set `fetch-depth: 1` by default which means it will only pull the current commit.